backend/modules/model_gateway/model_gateway.py

```diff
@@ -159,5 +177,22 @@ def get_reranker_from_model_config(self, model_name: str, top_k: int = 3):
             top_k=top_k,
         )
 
+    async def get_audio_model_from_model_config(self, model_name: str):
```

Comment:
This does not need to be async def, right?
I don't see any async call in this method itself

Reply:
It is used in `AudioParser` service.

Reply:
Is this because AudioProcessingSvc has been made into a async context manager?